vendor/bat-native-ledger/src/ledger_impl.cc

```diff
@@ -1115,6 +1112,10 @@ const confirmations::WalletInfo LedgerImpl::GetConfirmationsWalletInfo(
 
   wallet_info.payment_id = info.paymentId_;
 
+  if (info.keyInfoSeed_.empty()) {
```

Comment:
if you have corrupted state, this would fail, so we need to return blank wallet_info object

Reply:
should this be a blank wallet_info object, or an wallet with only the payment_id set?